_Group Projects/Group_2/Week_5/program.do

```diff
@@ -0,0 +1,60 @@
+/* Group 2
+Week 5 Assignment
+Authors: Felipe, Yuri, Moataz, and Yash 
+Last Modified: October 9, 2022 (09/10/2022)
+Program .do file */
+
+cd "/Users/devakid/Desktop/Git/econ490-fall22/_Group Projects/Group_2/Week_5"
+cap prog drop se_and_outlier
+prog define se_and_outlier 
+syntax anything
+
+preserve
+
+/* Identifying and dropping string variables before generating a covariance 
+matrix and mapping the distribution of standard errors
+*/
+ds, has(type string)
+drop `r(varlist)'
+
+// Generating a local for our dependent variable 
+local dep_var: word 1 of `anything'
+
+// Creating a list of the remaining non-string variables 
+ds `dep_var', not
+local othervar `r(varlist)'
+
+// Creating a loop to generate a covariance matrix for assessing collinearity issues
+foreach var in `othervar' {
+	corr, cov
+}
+
+/* Creating a loop to construct plots which check for the nature of the 
+distribution of the error terms
+*/ 	
+foreach var in `othervar' {
+	reg `dep_var' `var'
+	rvfplot, yline(0)
+	graph save `var'.gph, replace	
```

Comment:
Personally, I would prefer to save the graphs to the memory but not the disk. If you would also prefer this, I think you can change line 37 to

rvfplot, yline(0) name(`var')

and remove line 38.

Note that graphs from memory are displayed using "graph display" instead of "graph use"